dts/arm/st/stm32f446-pinctrl.dtsi

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2017 Pushpal Sidhu
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/pinctrl/stm32-pinctrl.h>
+
+/ {
+	soc {
+		pinctrl: pin-controller {
+			usart2_pins_b: usart2@1 {
```

Comment:
usart2 supported in stm32f4-pinctrl, remove these

Reply:
This is a different usart2 than the one supported in stmf4-pinctrl.

Reply:
USART2 node is defined in stm32f4.dtsi, file hence, if required, more combinations of usart2 could (and should) be added in stm32f4-pinctrl.
If we start spreading usart2 (or usartx) pin configurations across the various -pinctrl files, it'll soon be a mess to maintain.